daemon/README.md

```
@@ -0,0 +1,416 @@
+systemctl status cortexd
+
+# View logs
+journalctl -u cortexd -f
+
+# Test socket
+echo '{"method":"ping"}' | socat - UNIX-CONNECT:/run/cortex/cortex.sock
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)